### ISSUE TITLE
Relay message about join, part, nick change for everyone

### ISSUE THREAD

**Issue by acagastya** [FruitieX/teleirc#193]
`config.sendNonMsg = true;` sends IRC joins, parts and quits to Telegram -- but it relays only about the bot. If I change the nick, that information should be relayed as well.

Anyone who joined the channel, changed the nick, and left should be announced on Telegram group just like how IRC announces it in the chatrooms.

**Comment by Mikaela** [comment]
I think this is fixed by https://github.com/FruitieX/teleirc/commit/eb1b4652f221168321d7faa4ed2f08fe883a2c00 ?

**Comment by acagastya** [comment]
Nick changes are not relayed, yet.

**Comment by Mikaela** [comment]
@acagastya Which version of TeleIRC are you using? Does it include that comment, could you try to do the development install?

See [CONTRIBUTING.md](https://github.com/FruitieX/teleirc/blob/develop/CONTRIBUTING.md) for instructions.

**Comment by acagastya** [comment]
Yes, it contains an array of IRC events, join and part is working, but nick changes are not relayed.

**Comment by Varal7** [comment]
Giving my 2 cents here: not a bug, but a enhancement. 

```
    nodeIrc.on('nick', function(event) {
        if (!shouldRelayEvent('nick')) {
            return;
        }

        var chanName = event.channel;
        var user = event.nick;
        var newNick = event.new_nick;

        var channel = ircUtil.lookupChannel(chanName, config.channels);
        msgCallback({
            protocol: 'irc',
            type: 'nick',
            channel: channel,
            user: null,
            text: user + ' is now known as ' + newNick
        });
    });
```

Something like this would not work because event.channel is `undefined`. 
We'd have to send this notice to all chans where the user is.

**Comment by Varal7** [comment]
One way to achieve this would be to use [node-irc](https://github.com/martynsmith/node-irc) instead of [irc-framework](https://github.com/kiwiirc/irc-framework).
This might need some code rewriting, I still need to investigate how bad that would be.
